packages/noya-state/src/reducers/application.ts

```diff
@@ -1000,9 +1007,11 @@ export function reducer(
             : state.sketch.document.layerStyles?.objects ?? [];
 
         array.forEach((object: Sketch.Swatch | Sketch.SharedStyle) => {
-          if (ids.includes(object.do_objectID)) {
-            object.name = name;
-          }
+          if (!ids.includes(object.do_objectID)) return;
+          const group = object.name.split('/').slice(0, -1).join('/');
```

Comment:
Can this be simplified using the new `dirname` util?

Reply:
yes !